development/compiling/compiling_for_web.rst

```diff
@@ -8,9 +8,9 @@ Compiling for the Web
 Requirements
 ------------
 
-To compile export templates for the Web, the following is required:
+To compile Godot 3.4 export templates for the Web, the following is required:
```

Comment:
not too sure, but probably makes sense to give the context here for the required versions.

Reply:
No need, this is already in Godot 3.4 docs. If this becomes irrelevant in a future 3.x version, then we’ll remove it in a future version.